docs/sinco/v3.x/2_tutorials/1_restricting_permissions.md

```diff
@@ -0,0 +1,85 @@
+# Restricting Permissions
```

Comment:
should this file be prefixed with `7_` to be before the "Use Within Docker" tutorial? it shows up here:

<img width="283" alt="Screen Shot 2021-12-12 at 6 02 06 PM" src="https://user-images.githubusercontent.com/12766301/145733133-bc87da77-faff-4c53-98f0-5f2972830774.png">